frontend/e-buddy/src/components/Chat.tsx

```
@@ -24,67 +110,174 @@ export function Chat() {
+                {
+                  content: text || "Could not read response data",
+                  role: "assistant",
+                  id: traceId,
+                  display_type: "text",
+                },
+              ]);
+            };
+            reader.readAsText(event.data);
+            displayText = "Loading blob data...";
+          } else {
+            displayText = "Received non-text response: " + typeof event.data;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)